node/malus/integrationtests/0002-dispute-invalid-block.toml

```diff
@@ -0,0 +1,55 @@
+[settings]
+timeout = 1000
+
+[relaychain]
+default_image = "{{SYNTHIMAGE}}"
+chain = "wococo-local"
```

Comment:
Add a comment why we use `wococo-local` instead of `rococo-local` right now.

Reply:
Added 👍